WilliamsGarcia/26-08/HackerRank/InheritanceTwo/InheritanceTwo.java

```diff
@@ -0,0 +1,11 @@
+package com.company;
```

Comment:
Maybe you can re-arrange your classes into packages names like excercises.sampleone or bootcamp.dayone or so
What do you think??

Reply:
Sure, I'll work it. Ty